core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.css

```diff
@@ -63,7 +63,12 @@ oppia-tutor-card .oppia-learner-view-card.animate-card-width {
   transition: width 500ms;
 }
 
-oppia-tutor-card .oppia-learner-view-card-top-content p:not(:last-child) {
+.mat-checkbox p {
```

Comment:
@SanjaySajuJacob Wouldn't this apply for every single use of this rule throughout the app? This is why we typically try to restrict CSS rules in a particular file with a prefix.

/cc @aks681 @vojtechjelinek

Reply:
I added this CSS change in the tutor-card.component.css thinking that this change would apply only to the tutor component.

Reply:
Ah, I think you'll still need to prefix it. If not then it would apply to the whole page (I think this was an issue we ran into recently).

Reply:
Oh I see, I will try that. Thanks!